src/api.rs

```diff
@@ -528,11 +530,15 @@ impl Api {
                 )
             };
 
-            // TODO(kamil): Instead of breaking out of the loop here when we reach the limit
-            // (200 pages of 100 items, for a total of 20_000 files), we should send a list of hashes
-            // to the server, perform an intersection there, and change the upload mechanism
-            // to leave out only those files that were gave back to us by the server.
-            // This would also limit number of API requests for deduping from `N=[1:200]` to 1.
+            // Checksums need to be sorted in order to satisfy integration tests constraints.
+            if let Some(checksums) = checksums {
```

Comment:
I don't like it tbh, but didnt want to move sorting part to the caller.

Reply:
See my comment above, I think sorting can be done outside of this function.